craft_application/models/spread.py

```diff
@@ -72,6 +72,8 @@ class CraftSpreadYaml(SpreadBase):
         extra="forbid",
     )
 
+    project: str | None = None
+    path: str | None = None
```

Comment:
`project` is used, but why is `path` needed?

Reply:
(Per separate conversation: I added `path` because it matched what was missing in a test, but we shouldn't have it here)